internal/storage/data_codec_test.go

```diff
@@ -142,52 +142,52 @@ func TestInsertCodec(t *testing.T) {
 	insertData1 := &InsertData{
 		Data: map[int64]FieldData{
 			RowIDField: &Int64FieldData{
-				NumRows: 2,
+				NumRows: []int64{2},
```

Comment:
why change the type of numRows to slice?

Reply:
In order to know the size of each file, it is used to determine which file the vector to be searched for is in. 
For example, I know that the offset of the vector in the segment is 700. There are three files in this segment vector column, and the size of each file is 300, 300, 400, 500. I can know that it is in the third file. With numRows of []int64{300,300,400,500}. If I use the previous numRows, I can only know that the segment is 1500 rows in total. But can't know which file the vector I need is in